docs/source/index.rst

```diff
@@ -172,21 +172,60 @@ Furthermore, ``S3FileSystem.current()`` will return the most-recently created
 instance, so this method could be used in preference to the constructor in
 cases where the code must be agnostic of the credentials/config used.
 
-Self-hosted S3
---------------
+S3 Compatible Storage
+---------------------
 
-To use ``s3fs`` against your self hosted S3-compatible storage, like `MinIO`_ or
-`Ceph Object Gateway`_, you can set your custom ``endpoint_url`` when creating
-the ``s3fs`` filesystem:
+To use ``s3fs`` against an S3 compatible storage, like `MinIO`_ or
+`Ceph Object Gateway`_, you'll probably need to pass extra parameters when
+creating the ``s3fs`` filesystem. Here are some sample configurations:
 
-.. code-block:: python
+For a self-hosted minio instance:
 
+.. code-block:: python
+   # When relying on auto discovery for credentials
    >>> s3 = s3fs.S3FileSystem(
          anon=false,
```

Comment:
Can we also fix this example (to pass `anon=False`, instead of lowercase `false`)?

Reply:
Ha right ! Thanks ! I'll do that !